### ISSUE TITLE
Frontend Connection: Overview page of digital identities

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#56]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* DIs are displayed on overview page

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Backend is done. See #70 for the commits/the PR.

**Comment by Idontker** [comment]
Finished in sprint 5: 8b9cdf974b0592bb76a4d87a3d35fe9834c38272
